setup.md

```diff
@@ -3,17 +3,18 @@ layout: page
 title: Setup
 ---
 
-> ## Data
-> Data for this lesson is from the Portal Project Teaching Database -
-> [available on FigShare](https://figshare.com/articles/Portal_Project_Teaching_Database/1314459).
+> ## Datos
+> Los datos para esta lección son del **Portal Project Teaching Database** -
```

Comment:
```suggestion
> Los datos para esta lección son del **Proyecto de bases de datos de enseñanza** -
```

Reply:
Hmm. This one is a an official name. That's why I put it in bold and didn't translate it. 
![screen shot 2018-11-14 at 3 43 20 pm](https://user-images.githubusercontent.com/7544197/48514493-0a0eaa00-e824-11e8-99d4-2b10961168a4.png)

Reply:
OK, no sabia. Vale la pena poner el URL?

Reply:
Verdad. El URL sigue  "Los datos para esta lección son del **Portal Project Teaching Database** -
 [disponible en **FigShare**](https://figshare.com/articles/Portal_Project_Teaching_Database/1314459)."